apps/prairielearn/src/lib/score-upload.ts

```diff
@@ -286,7 +286,7 @@ function getFeedbackOrNull(json: Record<string, any>): Record<string, any> | nul
     } catch (e) {
       throw new Error(`Unable to parse "feedback_json" field as JSON: ${e}`);
     }
-    if (feedback_obj == null || !_.isPlainObject(feedback_obj)) {
+    if (feedback_obj == null || typeof feedback_obj !== 'object') {
```

Comment:
This and similar diffs are actually changes in behavior. The previous implementation would reject arrays, but this will not.

We may want to reach for something like https://github.com/sindresorhus/is-plain-obj? It's already in our dependency tree in a few places.

Reply:
Done in 93a4861a13adad0b221e3db45b842f1db2c36b2c. Also fixed a bug there (the logic for null was inverted). I just tested the score upload process to ensure this is working as expected.